manifest_staging/charts/csi-secrets-store-provider-azure/templates/arc-monitoring-config.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.arc.isMonitoringEnabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "sscdpa.arc.fullname" . }}-telegraf-config
```

Comment:
How do we handle the case when `{{ template "sscdpa.arc.fullname" . }}` is 63 chars? Appending `-telegraf-config` suffix will cause failures because length is > 63. Should we use static names for these?